x-pack/legacy/plugins/spaces/server/lib/request_interceptors/on_request_interceptor.test.ts

```diff
@@ -11,10 +11,12 @@ import {
   HttpServiceSetup,
```

Comment:
> Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

Hmm, maybe they weren't picked up by `jest_integration` for some reason? Are they run when you use `jest_integration` locally?

Reply:
hmm, apparently not:
```
elastic-mbp:x-pack larry$ node scripts/jest_integration.js ./legacy/plugins/spaces/server/lib/request_interceptors/
/Users/larry/repos/kibana/x-pack/scripts/jest_integration.js:19
throw new Error(`
^

Error: 
  jest_integration tests have been disabled because of a flaky failure in the
  example integration test: https://github.com/elastic/kibana/issues/32795#issuecomment-471585274

  when un-skipping these tests make sure to uncomment test/scripts/jenkins_xpack.sh lines 33-37

    at Object.<anonymous> (/Users/larry/repos/kibana/x-pack/scripts/jest_integration.js:19:7)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
```

Reply:
Ugh, didn't know that :sob: Sorry for wasting you time on that

Reply:
No worries! I'll just revert it